apps-aarch64.filesize

```diff
@@ -0,0 +1,10 @@
+5808	target/acap/bounding_box_example_0_0_0_aarch64.eap
```

Comment:
Are these sizes in KB?

Reply:
KiB I think:

> Display values are in units of the first available SIZE from --block-size, and the DU_BLOCK_SIZE, BLOCK_SIZE and BLOCKSIZE environment variables.  Otherwise,  units  default  to  1024
       bytes (or 512 if POSIXLY_CORRECT is set).

Reply:
I see, thanks!